sections/Dataset.tex

```diff
@@ -0,0 +1,10 @@
+\section{Dataset}
+Nordjyske is a media group maintaining many local newspapers, radios and websites in the North Jutland region of Denmark.
+All articles produced by Nordjyske are saved in a database.
+Each article is represented with a \texttt{pdf} file and an \texttt{xml} file, as well as one \texttt{png} file for each image in the article.
+The data set we are using is only a subset of all articles contained in the database.
```

Comment:
```suggestion
The dataset we are using is only a subset of all articles contained in the database.
```
Vi har ikke besluttet os for den ene eller anden måden at skrive dataset, men jeg kan personligt best lide denne.